src/track/trackrecord.h

```diff
@@ -87,6 +87,12 @@ class TrackRecord final {
             const QString& keyText,
             track::io::key::Source keySource);
 
+    // Merge the current metadata from the file with the subset of metadata
+    // that is stored in the library. This is required to delete any tags
+    // from the file that are not (yet) stored in the library database!
```

Comment:
Do we really delete user data from the users file, because we do not support it?

Reply:
The comment is wrong, the word "NOT" got lost somehow :see_no_evil:

Reply:
Ah, than it makes sense. :-)